src/Components/calculator.js

```diff
@@ -1,19 +1,18 @@
-/* eslint-disable react/destructuring-assignment */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/prefer-stateless-function */
 /* eslint-disable max-classes-per-file */
 import React from 'react';
```

Comment:
- [ ] It is a requirement that you should import `calcutate.js` into the `calculator.js` component. Kindly do so.

![image](https://user-images.githubusercontent.com/81932478/189329359-ab78f205-c2a8-405e-ad6f-d1ae38fdfa75.png)

Reply:
@Munsa1 are u sure that this is required for it to function...
I mean the only place a function from `calcutate.js` gets called is in the App.js after lifting the state up ...